extensions/interactions/rulesSpec.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests to check that all the relevant rule specs exist.
+ */
+
+describe('Rule spec services', function() {
+  var ruleSpecsServices = {};
+  var rulesJson;
+  var IMPLICIT_RULE_NAME = 'ClassifyMatches';
```

Comment:
This really ought to be ``CLASSIFY_RULE_NAME`` to stay consistent with implementation code.